qdev_wrappers/alazar_controllers/acquisition_parameters.py

```diff
@@ -37,10 +37,9 @@ def set_raw(self, value):
             value: value to set the parameter to
```

Comment:
this PR should also include a fix for [this `_save_val` call](https://github.com/qdev-dk/qdev-wrappers/blob/3412bc01e13060ea88428a876477700d9e0e5efb/qdev_wrappers/alazar_controllers/acquisition_parameters.py#L129). So far as i understand it, that call is just not needed (and was never needed in fact)